docs/docsite/rst/os_guide/intro_windows.rst

```diff
@@ -0,0 +1,173 @@
+.. _working_with_windows:
+
+Managing Windows hosts with Ansible
+===================================
+
+Managing Windows hosts is different from managing POSIX hosts. If you have managed nodes running Windows, review these topics.
+
+.. contents::
+   :local:
+
+This is an index of all the topics covered in this guide.
+
+.. toctree::
+   :maxdepth: 1
+
+   windows_dsc
```

Comment:
I noticed there are some 404s on this page: https://ansible--1820.org.readthedocs.build/projects/ansible-core/en/1820/os_guide/windows_dsc.html#ciminstance-type

While it's not part of this PR, it might be good to fix the broken links while we're in here.

References to class `MSFT_xWebAuthenticationInformation` and class `MSFT_xWebBindingInformation`

- Current link -> https://github.com/dsccommunity/xWebAdministration/blob/master/source/DSCResources/MSFT_xWebSite/MSFT_xWebSite.schema.mof
- Updated link -> https://github.com/dsccommunity/WebAdministrationDsc/blob/main/source/DSCResources/DSC_WebSite/DSC_WebSite.schema.mof

Reply:
Have updated the links and references to the DSC classes to reference the new type.